beesdoo_purchase/models/purchase.py

```diff
@@ -120,3 +120,15 @@ class PurchaseOrderLine(models.Model):
         help="Check this box to adapt the selling price "
         "on the product page when confirming Purchase Order",
     )
+
+    stock_coverage = fields.Float(compute="_compute_product", store=True)
+
+    @api.depends("product_id")
+    def _compute_product(self):
```

Comment:
`_compute_stock_coverage` would probably be a better name.

![](https://qqcitations.com/images-citations/citation-mal-nommer-les-choses-c-est-ajouter-au-malheur-du-monde-albert-camus-178378.jpg)